modules/monitoring-example-remote-write-authentication-settings.adoc

```diff
@@ -0,0 +1,284 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: REFERENCE
+[id="example-remote-write-authentication-settings_{context}"]
+= Example remote write authentication settings
+
+// The namespace name is the only difference between OCP and OSD/ROSA in the code blocks used in this reference module.
+ifndef::openshift-dedicated,openshift-rosa[]
+:namespace-name: openshift-monitoring
+:prometheus-instance: prometheusK8s
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+:namespace-name: openshift-user-workload-monitoring
+:prometheus-instance: prometheus
+endif::openshift-dedicated,openshift-rosa[]
+
+The following samples show different authentication settings you can use to connect to a remote write endpoint. Each sample also shows how to configure a corresponding `Secret` object that contains authentication credentials and other relevant settings. Each sample configures authentication for use with 
+ifndef::openshift-dedicated,openshift-rosa[]
+default platform monitoring
+endif::openshift-dedicated,openshift-rosa[]
+ifdef::openshift-dedicated,openshift-rosa[]
+monitoring user-defined projects
+endif::openshift-dedicated,openshift-rosa[]
+in the `{namespace-name}` namespace.
+
+.Sample YAML for AWS Signature Version 4 authentication
+====
+The following shows the settings for a `sigv4` secret named `sigv4-credentials` in the `{namespace-name}` namespace.
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: sigv4-credentials
+  namespace: {namespace-name}
+stringData:
+  accessKey: <AWS_access_key> <1>
+  secretKey: <AWS_secret_key> <2>
+type: Opaque
+----
+<1> The AWS API access key.
+<2> The AWS API secret key.
+
+The following shows sample AWS Signature Version 4 remote write authentication settings that use a `Secret` object named `sigv4-credentials` in the `{namespace-name}` namespace:
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
```

Comment:
for `openshift-user-workload-monitoring` project, configmap is `user-workload-monitoring-config`
for `openshift-monitoring` project, configmap is `cluster-monitoring-config`
from rosa doc: https://60762--docspreview.netlify.app/openshift-rosa/latest/monitoring/configuring-the-monitoring-stack.html#example-remote-write-authentication-settings_configuring-the-monitoring-stack,  the confimap is wrong for `openshift-user-workload-monitoring` project

Reply:
I changed all of these so that the examples in the OCP docs refer to the `cluster-monitoring-config` configmap, and the examples in the OSD and ROSA docs refer to the `user-workload-monitoring-config` configmap.